From 05691765b8bf19f9463d5028d957d6cac7b03463 Mon Sep 17 00:00:00 2001
From: Matthias Schoepfer <m.schoepfer@rethinkrobotics.com>
Date: Fri, 22 Sep 2023 10:20:06 +0200
Subject: [PATCH] CMakelists.txt: removing Werror

Some morron added Werror, which now turns deprecated warnings into
a failed compile. Thanks agian pal!

Signed-off-by: Matthias Schoepfer <m.schoepfer@rethinkrobotics.com>
---
 CMakeLists.txt | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index 61c6d81..c69a985 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -46,7 +46,7 @@ if(MSVC)
   target_compile_options(ruckig PRIVATE /W4 /WX)
 
 else()
-  target_compile_options(ruckig PRIVATE -Werror -Wall -Wextra)
+  target_compile_options(ruckig PRIVATE -Wall -Wextra)
 endif()
 
 
-- 
2.41.0

